src/offline.ts

```diff
@@ -0,0 +1,48 @@
+import { refetchCurrentUser, User } from '@openmrs/esm-framework';
+import Dexie, { Table } from 'dexie';
+import { postUserPropertiesOnline } from './components/choose-locale/change-locale-online.resource';
```

Comment:
This import is now the only remaining ugly part. Should we keep it that way or move the online API call to a shared location (where, in that case?)?

Reply:
Hm why is the offline part making this request?

I think this is part of the sync strategy, isn't it? I am not sure if it should be like that.

Maybe we can have a call about that later / after lunch.

Reply:
Yes exactly, it's part of syncing. Having a call is a good idea, I'm also free right now (if you still have 2 mins).